Misc/NEWS

```diff
@@ -47,6 +47,9 @@ Core and Builtins
 Library
 -------
 
+- bpo-30119: ftplib.FTP.putline() now throws ValueError on commands that contains
```

Comment:
We no longer update Misc/NEWS but now use the new blurb tool to create individual news entries in Misc/NEWS.d.  See https://devguide.python.org/committing/#what-s-new-and-news-entries